tools/ci_build/build.py

```diff
@@ -481,6 +481,7 @@ def convert_arg_line_to_args(self, arg_line):
         "--use_tvm_hash", action="store_true", help="Build ipp-crypto for hash generation. It is used by TVM EP only"
     )
     parser.add_argument("--use_tensorrt", action="store_true", help="Build with TensorRT")
+    parser.add_argument("--use_tensorrt_parser", action="store_true", help="Use TensorRT prebuilt parser")
```

Comment:
should we rename to use_tensorrt_builtin_parser to make it more clear?
in all cases, we do link against tensorrt parser, but in one case we build from source and the new option allows us to link against the parser binary included with tensorrt?
also update the help comment and replace prebuilt with builtin  (that makes it clear where the binary parser comes from)